### PR TITLE
I added changes based on #19

### LINKED ISSUE #19: Fix typo error in "Success Mantras"
**Describe the bug**

There's a typing error in the highlighted text view. 
Replace the highlighted textview by the following text - "Body and Mind are closely linked. When your body feels better so, too, will your mind."

**Would you like to work on the issue?**

- [ ] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by PratyushSingh07**
# Description

It solves the typing error in success mantra.

Fixes #19 

## Screenshot/APK Link

[Screenshot of the rectification](https://drive.google.com/file/d/1-5mWYNyAIoIBQ1iAY9IrZqiXtLfqpoAj/view?usp=sharing)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by coder2699** [comment]
@PratyushSingh07 in the screenshot text is not properly coming in the UI. Please fix that as well.

File: .idea/gradle.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="testRunner" value="PLATFORM" />
+        <option name="testRunner" value="GRADLE" />
```
Comment (coder2699) [inline .idea/gradle.xml:7]: All the changes made in .idea/gradle.xml files are system generated. Please don't push these changes.

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,5 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="DesignSurface">
```
Comment (coder2699) [inline .idea/misc.xml:3]: All the changes made in .idea/misc.xml files are system generated. Please don't push these changes.

**Comment by coder2699** [comment]
give @PratyushSingh07 5 points

**Comment by coder2699** [comment]
give @PratyushSingh07 5 points

**Comment by Harshpatel2910** [comment]
give @PratyushSingh07 5 points

**Comment by coder2699** [comment]
give @PratyushSingh07 5 points

**Pull request merged**